chsdi/templates/htmlpopup/cadastralwebmap.mako

```diff
@@ -6,7 +6,7 @@
 <%def name="table_body(c, lang)">
     <%
       from chsdi.models.vector import getScale
-      from chsdi.lib.validation.features import IdentifyServiceValidation
+      from chsdi.lib.validation.identify import IdentifyServiceValidation
```

Comment:
yes, it looks like the removed code was recently adjusted, so should I move this code from the features.py in the identify.py?